docs/application/native/guides/multimedia/webrtc.md

```diff
@@ -0,0 +1,447 @@
+    ...
+    /* In case of an answerer */
+    ret = webrtc_set_local_description(webrtc, answer_desc);
+    ... or ...
+    /* In case of an offerer */
+    ret = webrtc_set_remote_description(webrtc, answer_desc);
+    ...
+    /* If the connection is established successfully, you'll get notified of WEBRTC_STATE_PLAYING by _state_changed_cb() */
+    ```
+6. To get notified of various negotiation states, set callbacks by using `webrtc_set_peer_connection_state_change_cb()`, `webrtc_set_signaling_state_change_cb()`, `webrtc_set_ice_gathering_state_change_cb()` and `webrtc_set_ice_connection_state_change_cb()` function:
+
+    ```c
+    void _peer_connection_state_change_cb(webrtc_h webrtc, webrtc_peer_connection_state_e state, void *user_data)
+    {
+        /* After setting both description and ICE candidates from the remote peer, it'll be changed from WEBRTC_PEER_CONNECTION_STATE_CONNECTING to WEBRTC_PEER_CONNECTION_STATE_CONNECTED. */
+    }
+
+    void _signaling_state_change_cb(webrtc_h webrtc, webrtc_signaling_state_e state, void *user_data)
+    {
+        /* After setting local or remote description it'll be changed to WEBRTC_SIGNALING_STATE_HAVE_LOCAL_OFFER or WEBRTC_SIGNALING_STATE_HAVE_REMOTE_OFFER respectively. Both descriptions are set, it'll be changed to WEBRTC_SIGNALING_STATE_STABLE. */
+    }
+
+    void _ice_gathering_state_change_cb(webrtc_h webrtc, webrtc_ice_gathering_state_e state, void *user_data)
+    {
+        /* When setting local description, it'll be changed to WEBRTC_ICE_GATHERING_STATE_GATHERING. If the gathering work is done, it'll be changed to WEBRTC_ICE_GATHERING_STATE_COMPLETE. */
+    }
+
+    void _ice_connection_state_change_cb(webrtc_h webrtc, webrtc_ice_connection_state_e state, void *user_data)
+    {
+        /* After setting both description and ICE candidates from the remote peer, it'll be changed from WEBRTC_ICE_CONNECTION_STATE_CHECKING to WEBRTC_ICE_CONNECTION_STATE_COMPLETED. */
+    }
+
+    void webrtc_func(void)
+    {
+        int ret;
+        webrtc_h webrtc;
+        ...
+        ret = webrtc_set_peer_connection_state_change_cb(webrtc, _peer_connection_state_change_cb, user_data);
+        ret = webrtc_set_signaling_state_change_cb(webrtc, _signaling_state_change_cb, user_data);
+        ret = webrtc_set_ice_gathering_state_change_cb(webrtc, _ice_gathering_state_change_cb, user_data);
+        ret = webrtc_set_ice_connection_state_change_cb(webrtc, _ice_connection_state_change_cb, user_data);
+        ...
+        ret = webrtc_start(webrtc);
+        /* Do negotiation */
+    }
+    ```
+
+<a name="media_render"></a>
+## Rendering audio/video data
+
+You can decide how to handle audio/video streaming data received from a remote peer by using functions provided in this API set. You can also render sending audio/video data on the local target device.
+
+1. To get notified of creation of audio or video track from a remote peer, use `webrtc_set_track_added_cb()` function:
```

Comment:
Please avoid using the word "function" when function is already declared.
```suggestion
1. To get notified of creation of audio or video track from a remote peer, use `webrtc_set_track_added_cb()`:
```

Reply:
Okay, done.